smoke-test/tests/cypress/cypress.json

```diff
@@ -6,7 +6,7 @@
   "projectId": "hkrxk5",
   "defaultCommandTimeout": 10000,
   "retries": {
-    "runMode": 2,
+    "runMode": 1,
```

Comment:
Why 1?

Reply:
I don't think there is a good reason to run the retries more than once. It just makes smoke test take more time when there is a failure. I have never seen smoke test fail 2 times and then pass. If it fails 2 time we probably need to check for why instead of re-running it again. This will reduce our run times a bit in case there is flakiness